<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Browser</class>
 <widget class="QWidget" name="Browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt SQL Browser</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QToolButton" name="backButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Назад</string>
       </property>
       <property name="icon">
        <iconset resource="plan.qrc">
         <normaloff>:/icons/back.ico</normaloff>:/icons/back.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Обновить</string>
       </property>
       <property name="icon">
        <iconset resource="plan.qrc">
         <normaloff>:/icons/refresh.ico</normaloff>:/icons/refresh.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="table">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="selectAction">
   <property name="text">
    <string>S&amp;elect</string>
   </property>
   <property name="toolTip">
    <string>Refresh Data from Database</string>
   </property>
  </action>
  <action name="revertAction">
   <property name="text">
    <string>&amp;Revert All</string>
   </property>
   <property name="toolTip">
    <string>Revert</string>
   </property>
  </action>
  <action name="submitAction">
   <property name="text">
    <string>&amp;Submit All</string>
   </property>
   <property name="toolTip">
    <string>Submit Changes</string>
   </property>
  </action>
  <action name="rowStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit on &amp;Row Change</string>
   </property>
   <property name="toolTip">
    <string>Commit on Row Change</string>
   </property>
  </action>
  <action name="manualStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit &amp;Manually</string>
   </property>
   <property name="toolTip">
    <string>Commit Manually</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete Row</string>
   </property>
   <property name="statusTip">
    <string>Deletes the current Row</string>
   </property>
  </action>
  <action name="insertRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Insert Row</string>
   </property>
   <property name="statusTip">
    <string>Inserts a new Row</string>
   </property>
  </action>
  <action name="fieldStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit on &amp;Field Change</string>
   </property>
   <property name="toolTip">
    <string>Commit on Field Change</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>table</tabstop>
 </tabstops>
 <resources>
  <include location="plan.qrc"/>
 </resources>
 <connections/>
</ui>
